src/main.cpp

```diff
@@ -74,10 +73,12 @@ enum Options
   UNSAFE,
   BTF,
   INCLUDE,
+  EMIT_ELF,
+  EMIT_LLVM,
 };
 } // namespace
 
-void usage()
+void usage(bool extraHelp = false)
```

Comment:
this isn't used right?

Reply:
Good catch, I was going to make these options hidden by default and available with --full-help or something, but figured showing everything was probably a better user experience, especially since we don't have that many command line options.

Reply:
yeah lets leave it ou